internal/persistence/types.go

```diff
@@ -34,11 +34,14 @@ var (
 	// HomeDir is the home directory of the api7 cloud.
 	HomeDir = filepath.Join(os.Getenv("HOME"), ".api7cloud")
 	// TLSDir is the directory to store TLS certificates.
-	TLSDir string
+	TLSDir        string
+	credentialDir string
 )
 
 // Init initializes the persistence context.
 func Init() error {
+	credentialDir = filepath.Join(HomeDir, "credentials")
```

Comment:
What's the difference with the old code?

Reply:
@starsz Just optimize the code base: one place to fetch the `HomeDir`. We just have two places to use `os.Getenv("HOME")` before, now we just have one.

Reply:
The variable `HomeDir` was not referenced before, but directly using `os.GetEnv()`. Now use `HomeDir` consistently as a prefix for other file directories.

Reply:
Got it.